includes/admin/upgrades/v3/class-data-migrator.php

```diff
@@ -27,9 +27,10 @@ class Data_Migrator {
 	 *
 	 * @since 3.0
 	 *
-	 * @param object $data Data to migrate.
+	 * @param object $data       Data to migrate.
+	 * @param string $type       The type of address this is.
 	 */
-	public static function customer_addresses( $data = null ) {
+	public static function customer_addresses( $data = null, $type = 'billing' ) {
```

Comment:
Do we need the `$type` parameter given that it's always `billing` at this point?

Reply:
@ashleyfae I'd like to include it so that when we write the upgrade routine for the Simple Shipping migration, we can hook in and use this same method to handle things.